### PR TITLE
feat: model by javascript

### PR DESCRIPTION
**Pull request by aakankshadhurandhar**

### CODE REVIEW

File: Modal/js/script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Get the modal
```
Comment (abhimishra01) [inline Modal/js/script.js]: commenting is good, but upto a certain extend

commenting what's obvious is not recommented as per clean code practice

Ref:- https://github.com/ryanmcdermott/clean-code-javascript#comments

File: Modal/js/script.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+var modal = document.getElementById("myModal");
+
+// Get the button that opens the modal
+var btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
```
Comment (abhimishra01) [inline Modal/js/script.js]: same as above

File: Modal/js/script.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+btn.onclick = function () {
+  modal.style.display = "block";
+};
+
+// When the user clicks on <span> (x), close the modal
+span.onclick = function () {
+  modal.style.display = "none";
+};
+
+// When the user clicks anywhere outside of the modal, close it
```
Comment (abhimishra01) [inline Modal/js/script.js:20]: this is ok

**Review by abhimishra01 — CHANGES_REQUESTED** [review]
some minor changes, rest lgtm

**Review by abhimishra01 — APPROVED** [review]

File: Modal/js/script.js [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
-// Get the <span> element that closes the modal
+var click_btn = document.getElementsByClassName("btn-click")[0];
+var text_display = document.getElementsByClassName("text-display")[0];
 var span = document.getElementsByClassName("close")[0];
 
-// When the user clicks the button, open the modal
 btn.onclick = function () {
   modal.style.display = "block";
 };
+click_btn.onclick = function () {
```
Comment (abhimishra01) [inline Modal/js/script.js:11]: a new line in btw line 10 and 11 should make code clean and more redable

Comment (aakankshadhurandhar) [reply Modal/js/script.js:11]: done

**Pull request merged**